extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js

```diff
@@ -173,6 +173,27 @@ describe('amp-ad-api-handler', () => {
               });
             });
       });
+
+      it('should resolve on timeout', () => {
+        const noContentCallbackSpy = sandbox.spy();
+        apiHandler = new AmpAdApiHandler(adImpl, adImpl.element,
+             noContentCallbackSpy);
+        let clock;
+        const beforeAttachedToDom = element => {
+          clock = sandbox.useFakeTimers();
```

Comment:
You mean the clock? I think as long as it'll work as long as it is initialized before `startUp`  also work. I'll double check.